### PR TITLE
Added options for JAVA code gen

### PR DESCRIPTION
**Pull request by parv0888**
added options for java code gen
```
option java_package = "com.concordium.grpc.v2";
option java_multiple_files = true;
```

### CODE REVIEW

File: v2/concordium/health.proto [hunk 1]
```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 
 package concordium.health;
+option java_package = "com.concordium.grpc.v2";
+option java_multiple_files = true;
```
Comment (EmilLa1) [inline v2/concordium/health.proto:5]: Please provide a small comment explaining what each of the actions does.

File: v2/concordium/service.proto [hunk 2]
```diff
@@ -3,6 +3,8 @@ syntax = "proto3";
 import "v2/concordium/types.proto";
 
 package concordium.v2;
+option java_package = "com.concordium.grpc.v2";
+option java_multiple_files = true;
```
Comment (EmilLa1) [inline v2/concordium/service.proto:7]: Same here.

File: v2/concordium/types.proto [hunk 3]
```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 
 package concordium.v2;
+option java_package = "com.concordium.grpc.v2";
+option java_multiple_files = true;
```
Comment (EmilLa1) [inline v2/concordium/types.proto:5]: Same here.

File: v2/concordium/health.proto (hunk 1 above)
Comment (parv0888) [reply v2/concordium/health.proto]: added comments

File: v2/concordium/service.proto (hunk 2 above)
Comment (parv0888) [reply v2/concordium/service.proto]: added comments

**Pull request merged**